README.md

```diff
@@ -1,9 +1,7 @@
 # Nodes Vapor Template
```

Comment:
The rest of the README needs to be adjusted as well. Things like EnvironmentKey, Postgres instead of MySQL etc.